kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionEditor.vue

```diff
@@ -1,21 +1,410 @@
+      >
+        <div class="number-question">
+          <div>
+            <KTextbox
+              ref="numQuest"
+              v-model="numberOfQuestions"
+              type="number"
+              :label="numberOfQuestionsLabel$()"
+            />
+          </div>
+          <div>
+            <div
+              :style="borderStyle"
+              class="group-button-border"
+            >
+              <KIconButton
+                icon="minus"
+                aria-hidden="true"
+                class="number-btn"
+                :disabled="numberOfQuestions === 1"
+                @click="numberOfQuestions -= 1"
+              />
+              <span
+                :style="dividerStyle"
+              > | </span>
+              <KIconButton
+                icon="plus"
+                aria-hidden="true"
+                class="number-btn"
+                @click="numberOfQuestions += 1"
+              />
+            </div>
+          </div>
+        </div>
+      </KGridItem>
+    </KGrid>
+
+    <KTextbox
+      v-model="descriptionText"
+      :label="optionalDescriptionLabel$()"
+      :maxlength="400"
+      :textArea="true"
+      class="description-ktextbox-style"
+    />
+
+    <hr :style="dividerStyle">
+
+    <div>
+      <h5
+        class="section-settings-heading"
+      >
+        {{ questionOrder$() }}
+      </h5>
+      <KGrid>
+        <KGridItem
+          :layout12="{ span: 6 }"
+          :layout8="{ span: 4 }"
+          :layout4="{ span: 2 }"
+        >
+          <KRadioButton
+            v-model="selectedQuestionOrder"
+            :label="randomizedLabel$()"
+            :value="true"
+            :description="randomizedOptionDescription$()"
+          />
+        </KGridItem>
+        <KGridItem
+          :layout12="{ span: 6 }"
+          :layout8="{ span: 4 }"
+          :layout4="{ span: 2 }"
+        >
+          <KRadioButton
+            v-model="selectedQuestionOrder"
+            :label="fixedLabel$()"
+            :value="false"
+            :description="fixedOptionDescription$()"
+          />
+        </KGridItem>
+      </KGrid>
+    </div>
+
+    <hr :style="dividerStyle">
+
+    <h5
+      class="section-settings-heading"
+    >
+      {{ quizResourceSelection$() }}
+    </h5>
+    <p>{{ numberOfSelectedResources$({ count: 4 }) }}</p>
+
+    <KRouterLink
+      appearance="raised-button"
+      :to="{ path: 'select-resources' }"
+      class="change-resource-button-style"
+      iconAfter="forward"
+    >
+      {{ changeResources$() }}
+    </KRouterLink>
+
+    <hr :style="dividerStyle">
+
+    <h5
+      class="section-order-style section-settings-heading"
+    >
+      {{ sectionOrder$() }}
+    </h5>
+
+
+    <DragContainer
+      v-if="sectionOrderList.length > 0"
+      :items="sectionOrderList"
+      @sort="handleSectionSort"
+    >
+      <transition-group>
+        <Draggable
+          v-for="(section,index) in sectionOrderList"
+          :key="index"
+        >
```

Comment:
I think we should put a `background: $themeTokens.surface` CSS value on this because otherwise, the background is transparent and you can see through it as you drag it making it kind of look/feel insubstantial in a weird way. (I just realized this while using the draggable stuff w/ the accordion)